felix/rules/endpoints.go

```diff
@@ -51,6 +51,7 @@ func (r *DefaultRuleRenderer) WorkloadEndpointToIptablesChains(
 	adminUp bool,
 	tiers []TierPolicyGroups,
 	profileIDs []string,
+	flowLogEnabled bool,
```

Comment:
if this guy was a global option to the renderer, you would not need to change the signature of every method. As you observed yourself above.

Feels like this PR should not expand on this baggage, especially for internal methods.

Reply:
Make sense. It makes it much cleaner. Thanks.